src/components/DateRangePicker.jsx

```diff
@@ -43,6 +43,7 @@ const defaultProps = {
   numberOfMonths: 2,
   showClearDates: false,
   disabled: false,
+  hidePickerOnClear: false,
```

Comment:
@ljharb and @majapw so is it final then the prop name would be `reopenPickerOnClearDates ` whose default value would be false?